packages/sui-ssr/server/config.js

```diff
@@ -7,7 +7,7 @@ const DEFAULT_VALUES = {
 }
 
 const ASYNC_CSS_ATTRS =
-  'rel="stylesheet" media="only x" as="style" onload="this.media=\'all\';var e=document.getElementById(\'critical\');if(e){e.parentNode.removeChild(e);}"'
+  'rel="stylesheet" media="print" as="style" onload="this.media=\'all\'"'
```

Comment:
I think the idea is to leave a non-valid media type... is that a problem?

Reply:
It could be, as I read on filament group page: https://www.filamentgroup.com/lab/load-css-simpler/#why-not-use-a-faux-media-attribute%3F

Reply:
Ok, I didn't understand it right then....